Security/Firewall/JWTListener.php

```diff
@@ -41,7 +41,7 @@ class JWTListener extends AbstractListener
     protected $authenticationManager;
 
     /**
-     * @var EventDispatcherInterface
+     * @var BackwardsCompatibleEventDispatcher
```

Comment:
This is a BC break given the property is protected. We can fix it by making the wrapper implement `EventDispatcherInterface` (proxying any method call to the inner dispatcher, see https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/EventDispatcher/LegacyEventDispatcherProxy.php for inspiration)